environment-dev.yml

```diff
@@ -38,13 +39,15 @@ dependencies:
   - xsdba >=0.3.0
   # Opt
   - nc-time-axis >=1.3.1
-  - pyarrow >=10.0.1
+  - openpyxl
```

Comment:
Why put `openpyxl` in optional dependencies? Is there an issue with it?

Reply:
Not really. It's more that it's a library used specifically for Excel support used in one function. If a user only cares about writing to CSV or ODS, it's irrelevant.

I can mark it mandatory again, no issue.

Reply:
Until it becomes problematic, I'd keep it as mandatory.